packages/expo-modules-autolinking/src/types.ts

```diff
@@ -207,6 +212,30 @@ export type RawModuleConfigApple = {
   debugOnly?: boolean;
 };
 
+export type RawAndroidProjectConfig = {
+  name?: string;
+
+  path?: string;
+
+  publication?: AndroidPublication;
+
+  modules?: string[];
+
+  gradleAarProjects?: AndroidGradleAarProjectDescriptor[];
```

Comment:
question: is that intentional to have `gradleAarProjects` (which supports multiple projects) in `RawAndroidProjectConfig` (which stands for a single project)?

Reply:
Yes, one single Android project can depend on multiple aar projects. This construction reflects the dependency chain.